app/models/user.rb

```diff
@@ -102,8 +102,8 @@ def setup_and_save_user
     save
   end
 
-  def tune_after_persisted(organization)
-    add_to_organization organization
+  def tune_after_persisted(organization, tag_list = [])
+    add_to_organization organization, tag_list
```

Comment:
those changes passing these tags around the code, can be fixed by https://github.com/coopdevs/timeoverflow/pull/626/files#r623289771